docs/site/docusaurus.config.js

```diff
@@ -56,13 +56,32 @@ const config = {
     // ....
     [
       "@graphql-markdown/docusaurus",
+      /** @type {import('@graphql-markdown/types').ConfigOptions} */
       {
-        schema:
-          "../../crates/iota-graphql-rpc/schema.graphql",
+        id:'devnet',
+        schema: "https://raw.githubusercontent.com/iotaledger/iota/refs/heads/devnet/crates/iota-graphql-rpc/schema.graphql",
         rootPath: "../content", // docs will be generated under rootPath/baseURL
-        baseURL: "references/iota-api/iota-graphql/reference",
+        baseURL: "references/iota-api/iota-graphql/reference/Devnet/",
```

Comment:
I prefer devnet. But no strong opinion 😆 
```suggestion
        baseURL: "references/iota-api/iota-graphql/reference/devnet/",
```